src/app.scss

```diff
@@ -104,6 +104,10 @@ p {
     .pf-c-card__title h2 {
         font-size: var(--pf-global--FontSize--2xl);
     }
+
+    .pf-c-tab-content:focus {
+        outline: none;
+    }
```

Comment:
@teclator, @imobachgs 

I added those lines because the tab content is _focusable_ (not sure, but probably because of the _role_, see https://www.w3.org/WAI/PF/aria/roles#tablist) and it looks a bit weird when it gets the focus in webkit based browsers

```css
:focus {
    outline: -webkit-focus-ring-color auto 1px;
}
```
<sub>^^^ from Chrome ` user agent stylesheet`</sub>

![Interfaces tab content focused](https://user-images.githubusercontent.com/1691872/100383142-d7a00600-3014-11eb-9b11-04e71b52606c.png)

IMHO, it make no sense to display such visual feedback for the whole tab content (perhaps I'm missing something), so forcing a `none` outline is reasonable to me.

Additionally, I found that now _it's possible_ to show the outline only when the element was focused with the keyboard using the `:focus-visible` property in webkit based browsers or `:-moz-focusring` in Firefox (see [CSS Tricks](https://css-tricks.com/the-focus-visible-trick/) and [The Chromium Blog](https://blog.chromium.org/2020/09/giving-users-and-developers-more.html). It is really interesting to avoid those borders around certain actions when using the mouse.

|Selecting a tab | Expanding a row | Changing the status |
| - | - | - |
| ![Focus outline in the tab link](https://user-images.githubusercontent.com/1691872/100384235-96f5bc00-3017-11eb-95a6-b9e22de1e96f.png) | ![Focus outline in the toggle icon](https://user-images.githubusercontent.com/1691872/100384247-9ceb9d00-3017-11eb-8921-0f481c7e2ad6.png) | ![Focus outline in the switch](https://user-images.githubusercontent.com/1691872/100384256-a248e780-3017-11eb-9d5b-65f1185ce2cb.png) |
 
However, I decided to not go ahead using it by now.

Reply:
For me looks reasonable, @imobachgs what do you think?